cristin-import/src/main/java/no/unit/nva/cristin/lambda/FilenameEvent.java

```diff
@@ -5,6 +5,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.net.URI;
 import java.util.Optional;
+import nva.commons.core.JacocoGenerated;
```

Comment:
This class will be removed and importRequest will be reused. So the process is that if an actor sends an ImportRequest to a folder this will be break into several ImportRequests (one per file in folder). This is more generic than having two almost identical classes that do the same thing.